wp-config-pantheon.php

```diff
@@ -88,3 +88,13 @@
             break;
     }
 }
+
+/**
+ * Resolves valhalla performance issue
+ * 
+ * If you experience dashboard errors after applying this update, check to see
```

Comment:
It won't be dashboard errors. It will be PHP errors in their site:
```
php > define('FS_METHOD', 'direct');
php > define('FS_METHOD', 'direct');
PHP Warning:  Constant FS_METHOD already defined in php shell code on line 1

Warning: Constant FS_METHOD already defined in php shell code on line 1
```
Did not run this through NGINX to see if those errors show up on the screen, if it's a WSOD, if it's in the logs, etc.

Would be good to determine that, then add a direct quote of `PHP Warning:  Constant FS_METHOD already defined` in the comment.